spring-integration-smb/src/main/java/org/springframework/integration/smb/session/SmbConfig.java

```diff
@@ -134,6 +147,50 @@ public boolean isUseTempFile() {
 		return this.useTempFile;
 	}
 
+	/**
+	 * Gets the desired minimum SMB version value for what the Windows server will allow
+	 * during protocol transport negotiation.
+	 *
+	 * @return one of SMB1, SMB202, SMB210, SMB300, SMB302 or SMB311
+	 * @since 1.2
+	 */
+	public DialectVersion getSmbMinVer() {
+		return this.smbMinVer;
+	}
+
+	/**
+	 * Sets the desired minimum SMB version value for what the Windows server will allow
+	 * during protocol transport negotiation.
+	 *
+	 * @param _smbMinVer one of SMB1, SMB202, SMB210, SMB300, SMB302 or SMB311
+	 * @since 1.2
+	 */
+	public void setSmbMinVer(DialectVersion _smbMinVer) {
```

Comment:
May we use full words: `setSmbMinVersion/setSmbMaxVersion`?
The same is applied for properties and getters.

Thanks

Reply:
Done! Thanks.